deny.toml

```diff
@@ -82,6 +82,12 @@ skip-tree = [
     # wait for metrics-exporter-prometheus to upgrade
     { name = "hashbrown", version = "=0.13.2" },
 
+    # wait for zebra-chain to upgrade
```

Comment:
zebra-chain is in this repository, so we can upgrade it in this PR. We want to keep this dependency in sync with the ECC version so it stays consensus-compatible:
https://github.com/ZcashFoundation/zebra/blob/f1ee502ddf2bf8ffcdb60dbabdc91885f3d723c9/zebra-chain/Cargo.toml#L67-L68

Reply:
Moved to ticket #6638